src/variables/components/VariableImportOverlay.tsx

```diff
@@ -68,7 +68,6 @@ class VariableImportOverlay extends PureComponent<Props> {
     }
 
     createVariableFromTemplate(template)
-    getVariables()
```

Comment:
`createVariableFromTemplate` now handles the `getVariables` part so no need to call it explicitly here.